pkg/migrator/metrics/metrics.go

```diff
@@ -32,20 +32,21 @@ var (
 
 // CoreMigratorMetrics instruments core migrator with prometheus metrics.
 type CoreMigratorMetrics struct {
-	migrated  *prometheus.CounterVec
-	remaining *prometheus.GaugeVec
+	objectsMigrated *prometheus.CounterVec
+	remaining       *prometheus.GaugeVec
```

Comment:
probably better to rename it to objectsRemaining or sth? previously both migrated and remaining are about objects, now 'migration' is added, better to tell remaining is about 'objects' in the name

Reply:
Good point. Updated.